client/src/components/DataFiles/DataFilesTable/DataFilesTable.js

```diff
@@ -10,20 +10,22 @@ import { useSelector, useDispatch } from 'react-redux';
 import { useTable, useBlockLayout } from 'react-table';
 import { FixedSizeList, areEqual } from 'react-window';
 import AutoSizer from 'react-virtualized-auto-sizer';
-import { Link } from 'react-router-dom';
+import { Link, useLocation } from 'react-router-dom';
 import { LoadingSpinner, SectionMessage } from '_common';
 import './DataFilesTable.scss';
 import './DataFilesTable.module.scss';
 
 // What to render if there are no files to display
 const DataFilesTablePlaceholder = ({ section, data }) => {
+  const { pathname } = useLocation();
   const dispatch = useDispatch();
   const system = useSelector(state => state.pushKeys.target);
   const loading = useSelector(state => state.files.loading[section]);
   const err = useSelector(state => state.files.error[section]);
   const modalRefs = useSelector(state => state.files.refs);
   const filesLength = data.length;
 
+  const isGDrive = pathname.includes('googledrive');
```

Comment:
Instead of using `pathname`, could we find a way to check the `api` portion of the route that's declared?